src/physics/ra_rrtmg_sw.f90

```diff
@@ -10529,13 +10529,13 @@ SUBROUTINE RRTMG_SWRAD(                                        &
 !           IF ( mp_physics == FER_MP_HIRES .OR. &
 !                mp_physics == FER_MP_HIRES_ADVECT) THEN
 !#endif
-!                  DO K=kts,kte
```

Comment:
I don't understand why uncommenting this should be necessary, shouldn't it get set on lines 10420-10500 above? This code could only be called if mp_physics=FER_MP_HIRES (or FER_MP_HIRES_ADVECT) I think you were right to comment them out initially.